src/storage/src/render/sources.rs

```diff
@@ -180,43 +93,10 @@ where
     // at the end of `src.optimized_expr`.
     //
     // This has a lot of potential for improvement in the near future.
-    match src.connector.clone() {
+    match source_desc.connector.clone() {
         // Create a new local input (exposed as TABLEs to users). Data is inserted
         // via Command::Insert commands. Defers entirely to `render_table`
-        SourceConnector::Local { .. } => {
-            let mut table = render_table(as_of_frontier, scope);
-
-            let table_state = match storage_state.table_state.get_mut(&src_id) {
-                Some(table_state) => table_state,
-                None => panic!(
-                    "table state {} missing for source creation at worker {}",
-                    src_id,
-                    scope.index()
-                ),
-            };
-
-            // Make the new local input reflect the latest table state, then add the
-            // local input to the table state.
-            {
-                let mut session = table.handle.session(table.capability.borrow().clone());
-                for (row, time, diff) in &table_state.data {
-                    let mut time = *time;
-                    time.advance_by(table_state.since.borrow());
-                    assert!(time >= *table.capability.borrow().time());
-                    session.give((row.clone(), time, *diff));
-                }
-            }
-            table.capability.borrow_mut().downgrade(&table_state.upper);
-
-            table_state.inputs.push(LocalInput {
-                handle: table.handle,
-                // Hand off our a `Weak` to the core capability, so that
-                // it is correctly dropped if the token is dropped.
-                capability: Rc::downgrade(&table.capability),
-            });
-
-            ((table.ok_collection, table.err_collection), table.token)
-        }
+        SourceConnector::Local { .. } => unreachable!(),
```

Comment:
```suggestion
        SourceConnector::Local { .. } | SourceConnector::Log => unreachable!(),
```

See https://github.com/MaterializeInc/materialize/pull/12637, sorry for the churn!

Also, looks like the comment is outdated now?

Reply:
thanks, I just rebased